packages/block-library/src/navigation/edit/unsaved-inner-blocks.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import { isEqual } from 'lodash';
```

Comment:
Seems we're moving away from `lodash` as much as possible. Not sure this applies? [Lodash: Replace _.isEqual() with fastDeepEqual](https://github.com/WordPress/gutenberg/pull/46200)

Reply:
Didn't know about that library being in use. Nice! Thanks.